pkg/services/auth/auth_token.go

```diff
@@ -404,6 +410,35 @@ func (s *UserAuthTokenService) rotatedAfterParam() int64 {
 	return getTime().Add(-s.Cfg.LoginMaxInactiveLifetime).Unix()
 }
 
+const (
+	concurrentSessionLimit    = 3
```

Comment:
Would rather have this as a setting in [[auth]](https://grafana.com/docs/grafana/latest/administration/configuration/#auth). WDYT?

Reply:
Let's wait for Mitch to ask about [this](https://github.com/grafana/grafana/pull/27655#discussion_r491961771).

But yup, I agree that moving this constant into a configuration variable could be a good idea 👌

Reply:
I think we shouldn't change default behavior when introducing new features. So I'd prefer to have two config options: one for enabling user session limit and second for setting limit, like:
```ini
[auth]
enable_user_session_limit = false
user_session_limit = 10
```
Or maybe even just one config option `user_session_limit` which is `0` by default and disables limit.

Reply:
Based on earlier setting like `login_cookie_name`, `login_maximum_inactive_lifetime_duration` I would suggest  `login_maximum_concurrent_user_sessions` that default to 0 (disabled).